test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -230,7 +254,16 @@ void ProtoConverter::visit(Expression const& _x)
 
 void ProtoConverter::visit(BinaryOp const& _x)
 {
-	switch (_x.op())
+	BinaryOp_BOp op = _x.op();
+
+	if ((op == BinaryOp::SHL || op == BinaryOp::SHR || op == BinaryOp::SAR) &&
+		!m_evmVersion.hasBitwiseShifting())
+	{
+		m_output << dictionaryToken();
+		return;
+	}
```

Comment:
If we visit a binary operation that is one of shift (left/right) or arithmetic right shift, check whether the targeted evm version has these opcodes. If not, output a pseudo-random dictionary token.